features/features/package_managers/bundler_spec.rb

```diff
@@ -21,18 +27,11 @@
   end
 
   specify 'works with git dependency running outside of the bundle' do
-    # Run license_finder outside any Bundler environment
-    ::Bundler.with_unbundled_env do
-      # Fake home to avoid polluting real home with git checkout
-      ENV['HOME'] = LicenseFinder::TestingDSL::Paths.projects.to_s
-      # Make running license_finder pick up the correct implementation
-      ENV['RUBYOPT'] = "-I#{LicenseFinder::TestingDSL::Paths.root}/lib"
```

Comment:
All of this was needed because bundler was "leaking" environment variables (`bundle exec`). The solution is to wrap everything in `with_unbundled_env` and installing the ruby gem after building it, so it's "external" to the Gemfile, which is expected for feature specs.